pyciemss/ouu/qoi.py

```diff
@@ -1,22 +1,16 @@
+from typing import List
+
 import numpy as np
 
 
-def scenario2dec_nday_average(
-    dataCube, contexts: list = ["I_obs"], ndays: int = 7
-) -> np.ndarray:
+def obs_nday_average_qoi(dataCube, contexts: List, ndays: int = 7) -> np.ndarray:
     """
     Return estimate of last n-day average of each sample.
     dataCube is is the output from a Pyro Predictive object.
     dataCube[VARIABLE] is expected to have dimension (nreplicates, ntimepoints)
     Note: last ndays timepoints is assumed to represent last n-days of simulation.
     """
     if contexts is not None:
```

Comment:
I think we can remove this `if` statement. `contexts` is not of type `Optional`, so it shouldn't be possible for it to be `None`. Also, if it is `None`, then line 16 will error.

Reply:
Also, can you rename `dataCube` to be `result` and set its type to be `Dict[str, torch.Tensor]`.

Reply:
I changed `dataCube` to `samples` to since it hasn't been processed using `prepare_interchange_dictionary`. I updated the other things you mentioned.